webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeUpdateActions.scala

```diff
@@ -256,13 +257,16 @@ case class CreateSegmentVolumeAction(id: Long,
 
   override def applyOn(tracing: VolumeTracing): VolumeTracing = {
     val newSegment =
-      Segment(id,
-              anchorPosition.map(vec3IntToProto),
-              name,
-              creationTime,
-              colorOptToProto(color),
-              groupId,
-              AdditionalCoordinate.toProto(additionalCoordinates))
+      Segment(
+        id,
+        anchorPosition.map(vec3IntToProto),
+        name,
+        creationTime,
+        colorOptToProto(color),
+        groupId,
+        AdditionalCoordinate.toProto(additionalCoordinates),
+        metadata = MetadataEntry.toProtoMultiple(metadata)
```

Comment:
Deduplication is missing here as well

```suggestion
        metadata = MetadataEntry.toProtoMultiple(MetadataEntry.deduplicate(metadata))
```